keyboards/system76/launch_1/launch_1.c

```diff
@@ -188,6 +190,15 @@ bool process_record_kb(uint16_t keycode, keyrecord_t *record) {
                 set_value_all_layers(level);
             }
             return false;
+        case KC_LCTL:
+            lctl_pressed = record->event.pressed;
+            break;
+        case KC_RCTL:
+            rctl_pressed = record->event.pressed;
+            break;
+        case KC_ESC:
+            if (record->event.pressed && lctl_pressed && rctl_pressed) system76_ec_unlock();
```

Comment:
Try this, it will not send a key press or release for esc while lctrl and rctrl are held, and will do the EC unlock when pressed

```
if (lctl_pressed && rctl_pressed) {
    if (record->event.pressed) system76_ec_unlock();
    return false;
}
```

Reply:
That did it! Amended in https://github.com/system76/qmk_firmware/pull/24/commits/59766296b16083e16e5d8a523ae241ac8b1fd193